### PR TITLE
Started Cleaning Up RuboCop Errors

### PR DESCRIPTION
**Pull request by jgamblin**
I worked through the rubocop todo file this afternoon and cleaned up all the errors I felt comfortable fixing.

### CODE REVIEW

File: tasks/expanse/lib/cloud_exposure_mapping.rb [hunk 1]
```diff
@@ -25,7 +27,7 @@ def field_mapping_for_cloud_exposures # rubocop:disable Metrics/MethodLength
                   target: "details",
                   proc: lambda { |x|
                           "Exposed App Server Software: #{x['firstObservation']['configuration']['applicationServerSoftware']}"
-                        }                                                      },
+                        }                                                      }
```
Comment (j-collier) [inline tasks/expanse/lib/cloud_exposure_mapping.rb:30]: This line could probably be formatted in a more readable way

**Comment by jgamblin** [comment]
Corrected all reporting cops.

**Review by tmgerhart — DISMISSED** [review]
Reviewed each file but spent most time looking at changes for csv2kdi.  Wow on the number of changes.  Have tested the one on the csv2kdi successfully.  Noticed that a lot of the changes were double quotes to single quotes.  I approve with the caveat that I did not physically test any outside the csv2kdi.

**Comment by tmgerhart** [comment]
Before this is merged, I have found an error that I am researching:

Traceback (most recent call last):
        4: from toolkit.rb:48:in `<main>'
        3: from /opt/app/toolkit/tasks/csv2kdi/csv2kdi.rb:76:in `run'
        2: from /opt/app/toolkit/tasks/base.rb:94:in `run'
        1: from /opt/app/toolkit/tasks/base.rb:94:in `each'
/opt/app/toolkit/tasks/base.rb:95:in `block in run': undefined method `includes' for :task:Symbol (NoMethodError)

**Comment by tmgerhart** [comment]
The .include is not a method for a symbol class.  The previous approach to doing an include is the only way I believe would work for the exact purpose (match? looks for whole phrase to match the whole symbol value).  See the previous approach for line 95:

if k =~ /key/ ||  k =~ /token/ || k =~ /secret/

**Comment by jgamblin** [comment]
Going to rework this PR with better documentation.

**Pull request closed without merging**